package.json

```diff
@@ -51,21 +51,23 @@
     "eslint-config-prettier": "^6.5.0",
     "eslint-config-ts-react-important-stuff": "^1.1.0",
     "eslint-plugin-prettier": "^3.1.1",
+    "eslint-plugin-react-hooks": "^2.2.0",
```

Comment:
This can be removed (see comment above about how it's already being installed and turned on)